From 033dfd30d5224c5c24cdcb2633c803324c979f35 Mon Sep 17 00:00:00 2001
From: Gijs Vermeulen <gijsvrm@gmail.com>
Date: Thu, 17 Jun 2021 22:46:29 +0200
Subject: [PATCH] secur32: Default to building with GnuTLS support on macOS, if
 present.

Signed-off-by: Gijs Vermeulen <gijsvrm@gmail.com>
Signed-off-by: Hans Leidekker <hans@codeweavers.com>
Signed-off-by: Alexandre Julliard <julliard@winehq.org>
(cherry picked from commit 4940d2ada276a5c6e9f855177ef54ce23490d36e)
Conflicts:
    dlls/secur32/schannel_gnutls.c
Signed-off-by: Michael Stefaniuc <mstefani@winehq.org>
---
 dlls/secur32/schannel_gnutls.c | 4 ++--
 dlls/secur32/schannel_macosx.c | 4 ++--
 2 files changed, 4 insertions(+), 4 deletions(-)

diff --git a/dlls/secur32/schannel_gnutls.c b/dlls/secur32/schannel_gnutls.c
index 784c1ffe6e8..21e6ac7eae3 100644
--- a/dlls/secur32/schannel_gnutls.c
+++ b/dlls/secur32/schannel_gnutls.c
@@ -42,7 +42,7 @@
 #include "wine/debug.h"
 #include "wine/unicode.h"
 
-#if defined(SONAME_LIBGNUTLS) && !defined(HAVE_SECURITY_SECURITY_H)
+#if defined(SONAME_LIBGNUTLS)
 
 WINE_DEFAULT_DEBUG_CHANNEL(secur32);
 WINE_DECLARE_DEBUG_CHANNEL(winediag);
@@ -1098,4 +1098,4 @@ void schan_imp_deinit(void)
     libgnutls_handle = NULL;
 }
 
-#endif /* SONAME_LIBGNUTLS && !HAVE_SECURITY_SECURITY_H */
+#endif /* SONAME_LIBGNUTLS */
diff --git a/dlls/secur32/schannel_macosx.c b/dlls/secur32/schannel_macosx.c
index d725c9d22c2..ba589274b62 100644
--- a/dlls/secur32/schannel_macosx.c
+++ b/dlls/secur32/schannel_macosx.c
@@ -39,7 +39,7 @@
 #include "secur32_priv.h"
 #include "wine/debug.h"
 
-#ifdef HAVE_SECURITY_SECURITY_H
+#if defined(HAVE_SECURITY_SECURITY_H) && !defined(SONAME_LIBGNUTLS)
 
 WINE_DEFAULT_DEBUG_CHANNEL(secur32);
 
@@ -1242,4 +1242,4 @@ void schan_imp_deinit(void)
     TRACE("()\n");
 }
 
-#endif /* HAVE_SECURITY_SECURITY_H */
+#endif /* HAVE_SECURITY_SECURITY_H && !SONAME_LIBGNUTLS */
--
2.31.1
